CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- **feat**: [Swap] ERC-20 Approval Flow by @0xAlec #761. This is a breaking change that removes the `onSubmit` functionality from the `SwapButton` component and adds an approval flow for swapping from ERC-20 tokens.
```

Comment:
@0xAlec don't need to write this, as it is automated. But instead add this info in the `changeset` file.

Reply:
moved to changeset